lib/fizzy/CMakeLists.txt

```diff
@@ -2,19 +2,14 @@
 # Copyright 2019-2020 The Fizzy Authors.
 # SPDX-License-Identifier: Apache-2.0
 
-set(INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include)
-
 add_library(fizzy)
 add_library(fizzy::fizzy ALIAS fizzy)
 target_compile_features(fizzy PUBLIC cxx_std_17)
-target_include_directories(fizzy PUBLIC ${INCLUDE_DIR})
+target_include_directories(fizzy PUBLIC ${FIZZY_INCLUDE_DIR})
 
 target_sources(
-    fizzy
-
-    PUBLIC
-    ${INCLUDE_DIR}/fizzy/fizzy.h
-
+    fizzy PRIVATE
+    ${FIZZY_INCLUDE_DIR}/fizzy/fizzy.h
```

Comment:
Why is it `PRIVATE` now?

Only single `PRIVATE` keyword is enough now for all sources?

Reply:
`PUBLIC` mean something different: public sources from lib A are added to all libs that link to A. This is quite exotic and confusing feature, but not a way to specify public headers to my understanding. `install()` complains if a public source is specified as a local path.

I should move this fix to #530.

Yes, all sources after `PRIVATE` are private. The same as in all `target_*` commands.

Reply:
Don't repeat `PRIVATE` then?

Reply:
Removed.